src/Domain/Insights/ForbiddenNormalClasses.php

```diff
@@ -24,8 +24,15 @@ public function getTitle(): string
      */
     public function getDetails(): array
     {
+        $nonFinalClasses = array_filter(
+            $this->collector->getConcreteNonFinalClasses(),
+            function ($file) {
+                return $this->shouldSkipFile($file) === false;
+            }
+        );
```

Comment:
Remove duplication (a little different here as it doesn't have `ARRAY_FILTER_USE_KEY`)